src/component/Book.js

```diff
@@ -2,37 +2,52 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/bookSlice';
+import '../Styles/Book.css';
 
 function Book({ books }) {
   const dispatch = useDispatch();
 
   return (
-    <div>
+    <div className="book-container display-flex">
       <ul>
         {books.map((book) => (
           <li key={book.id}>
             <div>
-              <div>
-                <p>{book.category}</p>
-                <h1>{book.title}</h1>
-                <p>{book.author}</p>
-                <div>
-                  <button type="submit">comments</button>
-                  <div className="border" />
-                  <button
-                    type="submit"
-                    onClick={() => dispatch(deleteBook(book.id))}
-                  >
-                    Remove
-                  </button>
-                  <div className="border" />
-                  <button type="submit">Edit</button>
-                </div>
+              <div className="book-left display-flex">
+                <p className="category-book">{book.category}</p>
+                <h2 className="title-book">{book.title}</h2>
+                <p className="author-book">{book.author}</p>
+                <ul className="features">
+                  <li type="submit">Comments</li>
+                  <li>
+                    <button
+                      className="remove-button"
+                      type="submit"
+                      onClick={() => dispatch(deleteBook(book.id))}
+                    >
+                      Remove
+                    </button>
+                  </li>
+                  <li type="submit">Edit</li>
+                </ul>
               </div>
             </div>
           </li>
         ))}
       </ul>
+      <div className="book-middle display-flex">
+        <h2 className="circle-percent">O</h2>
+        <div className="middle-div display-flex">
+          <p>100%</p>
+          <p>Completed</p>
+        </div>
+      </div>
+      <div className="book-right">
+        <h3>CURRENT CHAPTER</h3>
+        <p>Chapter 23</p>
+        <button type="button" className="btn-update" label="Update progress" />
+        <h3 className="update-progress">UPDATE PROGRESS</h3>
+      </div>
```

Comment:
- [ ] It would be better if you can make your cards look like the Zephin design, the cards should not be on the same row they should be seperated colums please check the design 👍🏿 

- [ ] Please each of the cards should be on seperate columns please see the design 👍🏿 

Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
![Bookstore CMS (1)](https://github.com/Obote/Book-store/assets/69638013/186d68c7-1bee-43f1-91d7-eef5e60ec556)  | ![style](https://github.com/Obote/Book-store/assets/69638013/de10e31d-cfea-4d3a-9f6e-326e5e88d8ce)